repos/keg-components/src/components/form/autocomplete/autocomplete.example.js

```diff
@@ -0,0 +1,44 @@
+import { action } from '@storybook/addon-actions'
+import { Autocomplete } from './autocomplete'
+import { View } from 'KegView'
+import React from 'react'
+
+const types = [
+  'American Pygmy',
+  { text: 'American Pygmy', key: '123' },
+  'Alpine',
+  'Alpine Alternate',
+  'Angora',
+  'Boer',
+  'Don',
+  'Danish Landrace',
+  'Dandara',
+  "Lance's Pet",
+  'Nigerian Dwarf',
+  'Oberhasli',
+  'Orobica',
+  'Saanen',
+  'Toggenburg',
+  'Russian White',
+  'Valais Blackneck',
+]
+
+export const Basic = () => {
+  return (
+    <View>
+      <p>Start typing to see options</p>
```

Comment:
Can you use our own `P` component. How can we expect others to use it, if we don't even use it?

Reply:
updated